renderer/reducers/invoice.js

```diff
@@ -168,9 +168,10 @@ export const receiveInvoices = ({ invoices }) => dispatch => {
  * @param {string} cryptoUnit Crypto unit (sats, bits, btc)
  * @param {string} memo Memo
  * @param {boolean} isPrivate Set to true to include routing hints
+ * @param {string} fallbackAddress on-chain address fallback
  * @returns {Function} Thunk
  */
-export const createInvoice = (amount, cryptoUnit, memo, isPrivate) => async (
+export const createInvoice = (amount, cryptoUnit, memo, isPrivate, fallbackAddress) => async (
```

Comment:
Consider refactoring to accept an `options` object. This is starting to make for a hard to read caller signature.

Reply:
good call